app/components/windows/Troubleshooter.vue

```diff
@@ -1,107 +1,112 @@
 <template>
-<modal-layout :showControls="false" :customControls="true">
-  <div slot="controls">
-    <start-streaming-button v-if="issue.code === 'FRAMES_DROPPED'"></start-streaming-button>
-    <button
-        class="button button--action"
-        @click="showSettings"
-    >
-      {{ $t('Open Settings') }}
-    </button>
-  </div>
-
-  <div slot="content">
-
-    <div v-if="issue.code === 'FRAMES_DROPPED'">
-      <h4>
-        <span class="fa fa-warning"></span>
-        {{ issue.message }}
-      </h4>
-      <p>
-        {{ $t(' Streamlabs has detected dropped frames') }} {{ moment(issue.date) }}.<br/>
-      </p>
-      <h4>{{ $t('What does this mean?') }}</h4>
-      <p>
-        {{ $t(' Some frames have not been uploaded. This problem is usually related to a poor network connection.') }}
-      </p>
-      <h4>
-        {{ $t('What can I do?') }}
-      </h4>
-
-      <ul>
-        <li>{{ $t('Check the health of your Internet connection') }}</li>
-        <li>{{ $t('Change your ingest server') }}</li>
-        <li>{{ $t('If none of these worked, lower your bitrate') }}</li>
-      </ul>
-
-      <div class="inline-controls">
-        <h4 v-if="isStreaming">
-          {{ $t('Stop streaming to access these controls:')}}
-        </h4>
-        <GenericFormGroups v-model="streamingSettings" @input="saveStreamingSettings" />
-        <GenericFormGroups v-model="outputSettings" @input="saveOutputSettings" />
-      </div>
+  <modal-layout :showControls="false" :customControls="true">
```

Comment:
Struggling to understand how these changes are relevant to this PR.  Is this just an indentation change on this unrelated file?

Reply:
hm yea im not sure when/how this happened but i think it's just a prettier autoformat